lib/authorizer/jwt.js

```diff
@@ -217,8 +219,18 @@ module.exports = {
                 payload = trimmedPayload && JSON.parse(trimmedPayload);
             }
 
+            payload = payload || {};
+
+            payload.iat = auth.get(AUTH_KEYS.IAT);
```

Comment:
@codenirvana Should we fail/throw if `payload.iat` is also set? Currently we're just overriding.

Reply:
@coditva we can do it like.
`payload.iat = payload.iat ?? auth.get(AUTH_KEYS.IAT)`